src/main/java/io/cdap/plugin/gcp/bigquery/sink/AbstractBigQuerySink.java

```diff
@@ -126,6 +145,36 @@ public final void onRunFinish(boolean succeeded, BatchSinkContext context) {
     }
   }
 
+  private void recordMetric(BatchSinkContext context, Job queryJob) throws Exception {
+    String operation = baseConfiguration.get(BigQueryConstants.CONFIG_OPERATION);
+    String metricName = String.format("records.%s", operation.toLowerCase());
+    long totalRows = getTotalRows(queryJob);
+    LOG.info(String.format("Total row count is %d" , totalRows));
```

Comment:
would be good to include the job id in this message. Also, logger can use '{}' as a placeholder instead of String.format().

`LOG.info("Job {} affected {} rows")`

Reply:
Done